### ISSUE TITLE
Some button pins in the wiring sketch are not correct

### ISSUE THREAD

**Issue by MakerMeik** [johan-m-o/LEDTennis#1]
Really cool game, thanks for sharing. My kids and I are very excited.
But I also found a little "bug". In the wiring diagram, the button for player 1 is on pin D13, but it is D2. And the "left" button is shown as D3, but it's actually D6.

**Comment by johan-m-o** [comment]
That's right! Thank you for pointing that out.

As stated in the text, I made that diagram before starting the build. At a later point I wanted to try using interrupts for the player buttons and that means using D2 and D3 on a Nano. I later found I did not need external interrupts at all, but by that point everything was soldered up already and I had basically forgotten about the original schematic.

So to summarise, player 1 is connected to D2, player 2 is connected to D3 and the left button is connected to D6.

I'll get that updated in the readme shortly...

Thank you for letting me know and I'm absolutely thrilled that someone would even consider building this

**Comment by johan-m-o** [comment]
That should be a better representation. Again, thanks.

**Comment by MakerMeik** [comment]
Wow ... that was quick! Thanks ...
I am sure that your project will bring joy to many more people. :-)
